[2024-02-01 17:59:33,843] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_25.dim_postgres scheduled__2024-01-31T17:59:18.936298+00:00 [queued]>
[2024-02-01 17:59:33,864] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_25.dim_postgres scheduled__2024-01-31T17:59:18.936298+00:00 [queued]>
[2024-02-01 17:59:33,864] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-01 17:59:33,864] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-01 17:59:33,865] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-01 17:59:33,900] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): dim_postgres> on 2024-01-31 17:59:18.936298+00:00
[2024-02-01 17:59:33,911] {standard_task_runner.py:52} INFO - Started process 10516 to run task
[2024-02-01 17:59:33,916] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'project_6_25', 'dim_postgres', 'scheduled__2024-01-31T17:59:18.936298+00:00', '--job-id', '389', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp5fdyld_e', '--error-file', '/tmp/tmpa6ejonrg']
[2024-02-01 17:59:33,918] {standard_task_runner.py:80} INFO - Job 389: Subtask dim_postgres
[2024-02-01 17:59:34,226] {task_command.py:369} INFO - Running <TaskInstance: project_6_25.dim_postgres scheduled__2024-01-31T17:59:18.936298+00:00 [running]> on host 1fafaef2f352
[2024-02-01 17:59:34,657] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project_6_25
AIRFLOW_CTX_TASK_ID=dim_postgres
AIRFLOW_CTX_EXECUTION_DATE=2024-01-31T17:59:18.936298+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-31T17:59:18.936298+00:00
[2024-02-01 17:59:34,775] {base.py:68} INFO - Using connection ID 'postgres_dwh' for task execution.
[2024-02-01 17:59:34,927] {dbapi.py:208} INFO - Running statement: CREATE TABLE IF NOT EXISTS case_table (
    id INTEGER,
    status_name VARCHAR(255) NOT NULL,
    status_detail VARCHAR(255) NOT NULL
);

CREATE TABLE IF NOT EXISTS province_table (
    province_id INTEGER PRIMARY KEY,
    province_name VARCHAR(255) NOT NULL
);

CREATE TABLE IF NOT EXISTS district_table (
    district_id INTEGER PRIMARY KEY,
    province_id INTEGER,
    district_name VARCHAR(255) NOT NULL
);
, parameters: None
[2024-02-01 17:59:34,969] {postgres.py:94} INFO - NOTICE:  relation "case_table" already exists, skipping

[2024-02-01 17:59:34,971] {postgres.py:94} INFO - NOTICE:  relation "province_table" already exists, skipping

[2024-02-01 17:59:34,971] {postgres.py:94} INFO - NOTICE:  relation "district_table" already exists, skipping

[2024-02-01 17:59:35,170] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=project_6_25, task_id=dim_postgres, execution_date=20240131T175918, start_date=20240201T175933, end_date=20240201T175935
[2024-02-01 17:59:35,265] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-01 17:59:35,834] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
